packages/webapp/src/members/components/members-grid.tsx

```diff
@@ -1,17 +1,22 @@
 import React from "react";
 import Link from "next/link";
+import dayjs from "dayjs";
+import * as localizedFormat from "dayjs/plugin/localizedFormat";
 
 import { atomicAssets, ipfsBaseUrl } from "config";
 import { MemberData } from "../interfaces";
 import { assetToString } from "_app";
 
+dayjs.extend(localizedFormat.default);
```

Comment:
is this dayjs supposed to exist in every file that we reference it or should we add this globally in our config.ts?

Reply:
I think you’re right; prob at the top level. I’ll play with it today.